pkg/suggestion/v1alpha2/katib_suggestion/algorithm_register.py

```diff
@@ -0,0 +1,8 @@
+from .random_search import RandomSearch
+from .grid_search import GridSearch
+from .bayesian_optimization_algorithm import BOAlgorithm
+
+
+ALGORITHM_REGISTER = {"random_search": RandomSearch,
```

Comment:
How would someone distinguish between an algorithm that is still in development and one that is stable? What if users want to enable or disable a specific algorithm?

Also if a user wants to add a new algorithm, how would they achieve this without changing the shared Docker image?

Reply:
can we make it configurable instead of hard-code